### PR TITLE
13 improve description

### LINKED ISSUE #13: Improve description
We need to improve the description of what each method does, right now what most of all say is: "Get balance of an account2"

### PR DESCRIPTION
**Pull request by ederaleng**
Updated the descriptions of each pool method

### CODE REVIEW

File: assembly/proto/core.proto [hunk 1]
```diff
@@ -96,7 +103,7 @@ message approve_arguments {
   uint64 value = 3 [jstype = JS_STRING];
 }
 
-// @description initializing core contract
+// @description get reserver of a pool
```
Comment (vonlooten) [inline assembly/proto/core.proto]: @ederaleng "get reserves of a pool" ?

File: assembly/proto/core.proto [hunk 2]
```diff
@@ -87,7 +94,7 @@ message transfer_arguments {
   uint64 value = 3 [jstype = JS_STRING];
 }
 
-// @description Get balance of an account 3
+// @description Sets `amount` as the allowance of `spender` over the caller's tokens.
```
Comment (vonlooten) [inline assembly/proto/core.proto]: @ederaleng Sets `value` as the allowance of `spender` over the caller's tokens. 

// because parameter name is value

**Review by vonlooten — CHANGES_REQUESTED** [review]
Looks good, I only had some questions regarding the descriptions

File: assembly/proto/core.proto (hunk 1 above)
Comment (ederaleng) [reply assembly/proto/core.proto]: This method returns the token reserves within the pool

File: assembly/proto/core.proto (hunk 2 above)
Comment (ederaleng) [reply assembly/proto/core.proto]: I just changed amount to value

**Review by vonlooten — APPROVED** [review]

**Pull request merged**